File: client/Assets/Scripts/Schema/CSVData/CSVData_String.cs
//////////////////////////////////////////////////////////////////////////
//
// 文件：Assets\Scripts\Schema\CSVData\CSVData_String.cs
// 作者：Xoen Xie
// 时间：2023/06/19
// 描述：CSV数据 string
// 说明：
//
//////////////////////////////////////////////////////////////////////////
using UnityEngine;
using System;

namespace XsfScp
{
    public sealed class CSVData_String : CSVData
    {
        public override CSVDataType DataType { get { return CSVDataType.String; } }

        public string sValue;

        public override void Read(int nRow, int nCol, string sData)
        {
            sValue = sData;
        }
    }
}

File: server-cpp/source/schema/src/csv_data/CSVData_String.cpp
//////////////////////////////////////////////////////////////////////////
//
// 文件：source/xsf/src/schema/csv_data/CSVData_String.cpp
// 作者：Xoen Xie
// 时间：2023/08/14
// 描述：CSV数据 字符串
// 说明：
//
//////////////////////////////////////////////////////////////////////////

#include "CSVData_String.h"
#include "XSFMallocHook.h"
#include "XSF.h"

namespace xsf_scp
{
    bool CSVData_String::Read(uint32 row, uint32 col, const char * sData)
    {
        sValue = sData;
        return true;
    }
}
